### DISCUSSION TITLE
Safari Erweiterung über App Store?

### DISCUSSION THREAD

**Discussion by tomillr** [stefanw/voebbot#477] [Ideas]
Gibt es Hoffnung, die bibot Erweiterung auch über den offiziellen App Store angeboten zu bekommen? Es beunruhigt mich nämlich ein wenig, mit der "Erlaube Unsignierte Erweiterungen" einen Einfall-Vektor für bösartigen Code (einen Key-Logger z.B.) aufzumachen.

**Comment by Maik-Wi** [comment]
Die Erweiterung kommt vermutlich nicht durch den Review-Prozess von Apple.
